jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java

```diff
@@ -44,6 +46,16 @@ public void circlePaginationTest() {
             circlePagination.has().value(page);
         }
         circlePagination.is().ended();
+        circlePagination.is().circle();
```

Comment:
Please divide tests by features (for example, one for circle feature, one to check theme, one to check color, ....) and not duplicate checks